vision_agent/utils/video_tracking.py

```diff
@@ -269,10 +255,26 @@ def post_process(
         Dict[str, Any]: Post-processed data including return_data and display_data.
     """
     return_data = []
-    for frame_idx, frame in enumerate(merged_detections):
+    label_remapping = {}
+    for _, frame in enumerate(merged_detections):
         return_frame_data = []
         for detection in frame:
-            label = f"{detection['id']}: {detection['label']}"
+            label = detection["label"]
+            id = detection["id"]
+
+            # Remap label IDs so for each label the IDs restart at 1. This makes it
+            # easier to count the number of instances per label.
+            if label not in label_remapping:
+                label_remapping[label] = {"max": 1, "remap": {id: 1}}
+            elif label in label_remapping and id not in label_remapping[label]["remap"]:  # type: ignore
+                max = label_remapping[label]["max"]
+                max += 1  # type: ignore
+                label_remapping[label]["remap"][id] = max  # type: ignore
+                label_remapping[label]["max"] = max
```

Comment:
can we use `max_id` instead of `max` so we don't conflic the built-in `max` python function? 
```suggestion
                max_id = label_remapping[label]["max"]
                max_id += 1  # type: ignore
                label_remapping[label]["remap"][id] = max  # type: ignore
                label_remapping[label]["max"] = max_id
```

Reply:
good call